Trisha-tech/OnlineBookSales
# Fetch trending books from booksapi

### Description
Fetches latest trending Newyork times best sellers from the NY_TIMES_BOOK_API.

### Related Issues

- Placeholder: "Fixes #353 "

### Changes
List the detailed changes made in this PR.
- Added improvents to trending section.
- Added a index.js file client/src/Components ( because it was not available before and the project was giving error) 
  So index.js in Components fixed those errors.
- Added a .env file (.env.sample) in client directory ( To access REACT_APP_NY_TIMES_API_KEY in Trending.jsx)

### Testing Instructions
Detailed instructions on how to test the changes. Include any setup needed and specific test cases.
1. Pull this branch.
2. Create a .env file in client directory. (see .env.sample )
3. Get the NY_TIMES_API_KEY from https://developer.nytimes.com/ ( Create and accont and get the books apikey)
4. Add this apiKey to your .env file.
5. npm install  ( in client)
6. npm start   (in client )

### Screenshots (if applicable)
Add any screenshots that help explain or visualize the changes.


### Screenshots (if applicable)
![Screenshot from 2024-10-09 00-22-27](https://github.com/user-attachments/assets/a5e80517-ef9d-43ea-8663-3f5a0a1cad4d)

### Additional Context
See issue #353  for more info

### Checklist
Make sure to check off all the items before submitting. Mark with [x] if done.
